### PR TITLE
Feat: NTRN-29. Add global fee module

### PR DESCRIPTION
**Pull request by albertandrejev**
Test for the globalfees module on the Netron.

https://hadronlabs.atlassian.net/browse/NTRN-29

Related PRs:
https://github.com/neutron-org/neutron/pull/264
https://github.com/neutron-org/neutron-docs/pull/91

### CODE REVIEW

**Review by pr0n00gler — APPROVED** [review]

**Review by foxpy — APPROVED** [review]

File: src/testcases/run_in_band/globalfee.test.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    neutronAccount = new WalletWrapper(
+      neutronChain,
+      testState.wallets.qaNeutron.genQaWal1,
+    );
+    const daoCoreAddress = (await neutronChain.getChainAdmins())[0];
+    const daoContracts = await getDaoContracts(neutronChain, daoCoreAddress);
+    dao = new Dao(neutronChain, daoContracts);
+    daoMember = new DaoMember(neutronAccount, dao);
+    await daoMember.bondFunds('1000');
+    await getWithAttempts(
```
Comment (sotnikov-s) [inline src/testcases/run_in_band/globalfee.test.ts:36]: what's the point of getting with attempts here? bondFunds uses executeContract inside it which is a sync op, so the bonded funds must become accrued voting power just after the bondFunds. getting with attempts in such case would only complicate code and increase time consumed by the test in a failing case here

File: src/testcases/run_in_band/globalfee.test.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      async () =>
+        await dao.queryVotingPower(daoMember.user.wallet.address.toString()),
+      async (response) => response.power == 1000,
+      20,
+    );
+
+    await daoMember.user.msgSend(dao.contracts.core.address, '1000', {
+      gas_limit: Long.fromString('200000'),
+      amount: [{ denom: daoMember.user.chain.denom, amount: '500' }],
+    });
```
Comment (sotnikov-s) [inline src/testcases/run_in_band/globalfee.test.ts:47]: why this send?

File: src/testcases/run_in_band/globalfee.test.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  });
+
+  afterAll(async () => {
+    await daoMember.unbondFunds('1000');
```
Comment (sotnikov-s) [inline src/testcases/run_in_band/globalfee.test.ts:51]: why unbond?

**Review by sotnikov-s — APPROVED** [review]

File: src/testcases/run_in_band/globalfee.test.ts (hunk 1 above)
Comment (albertandrejev) [reply src/testcases/run_in_band/globalfee.test.ts:36]: It was added there to be fully sure that bond funds was successfully bonded t will throw exception if something goes wrong.

File: src/testcases/run_in_band/globalfee.test.ts (hunk 2 above)
Comment (albertandrejev) [reply src/testcases/run_in_band/globalfee.test.ts:47]: It was taken from `governance.tsst.ts` test, and I forgot to remove it from current test.

**Pull request merged**